### PR TITLE
fix: branch release for open-event-attendee

### LINKED ISSUE #2483: Fix Branch Release

### PR DESCRIPTION
**Pull request by adityastic**
Fixes #2483

### CODE REVIEW

**Comment by adityastic** [comment]
@iamareebjamal should fix the branch release

File: scripts/update-apk.sh [hunk 1]
```diff
@@ -69,6 +69,7 @@ if $IS_PUBLISH_BRANCH ;then
     if $PR_FOR_RELEASE ;then
         exit 0
     fi
+    cd apk
```
Comment (iamareebjamal) [inline scripts/update-apk.sh]: Can't we just remove `cd ..` above and change paths accordingly

Comment (adityastic) [reply scripts/update-apk.sh]: @iamareebjamal i didn't get what you meant by changing paths accordingly

Comment (iamareebjamal) [reply scripts/update-apk.sh]: The path of apk and fastlane json

Comment (adityastic) [reply scripts/update-apk.sh]: Oh yes, we can definitely do that. Let me update the pr

**Pull request merged**